ballsdex/settings.py

```diff
@@ -132,6 +135,8 @@ def read_settings(path: "Path"):
     settings.prometheus_port = content["prometheus"]["port"]
 
     settings.max_favorites = content.get("max-favorites", 50)
+
+    settings.rarities = content.get("rarities", {})
```

Comment:
```suggestion
    settings.rarities = content.get("rarities", {})
    settings.currency_name = content["currency-name"]
```
without this, having settings.currency_name in the config wouldn't do anything